task_2/3.js

```diff
@@ -0,0 +1,63 @@
+// Функция принимает массив чисел. Возвращает массив строк такого вида:
+// fn([3,4,6]); // ['Value: 3; Index: 0; Squared: 9','Value: 4; Index: 1; Squared: 16','Value: 6; Index: 2; Squared: 36']
+
+//1
+const squaredNumbers1 = (numbers) => {
+    const result = [];
+
+    for(let i = 0; i < numbers.length; i++) {
+        if(numbers[i] === 0) {
```

Comment:
1. same as above. create separate variable for access numbers[i]. fix it in all places
2. 0 ** 2 will be 0, no need to handle this case